how-to-contribute-to-workshops/how-to-contribute-to-workshops.md

```diff
@@ -0,0 +1,60 @@
+# How to Contribute to Workshops
+
+## Basic Requirements for Integrating Any Workshop
+
+For any workshop that you wish to contribute, ensure that it is sufficiently distinct from the workshops that Bit Project currently has. See <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization#step-1-workshop-proposal">Workshop Proposal</a> for the complete list of the workshops Bit Project currently has.
+
+Successful integration of any workshop involves the following main stages and sub-stages:
+
+* **Placing your work in GitBook**
+* **Stage 1: Preparation**
+  * Step 1: Workshop Proposal
+  * Step 2: Interactive Component Idea
+  * Step 3: Responsibilities Delegated
+* **Stage 2: Slides and Components**
+  * Step 1: Presentation Slides Outline
+  * Step 2: Completing Slides and Interactive Component
+* **Stage 3: Presentation**
+  * Step 1: Presentation Practice
+  * Step 2: Present to Bit & Organization
+
+For detailed information about each step and sub-step of the above workflow, see <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization">Joining as an Organization</a>.
+
+<h2 id = "Porting">Porting an Existing Workshop Into Our System
+
+If you wish to contribute an existing workshop, ensure that all requirements outlined in <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization">Joining as an Organization</a> have been met and make edits to your workshop as neccessary. 
+
+You should be writing the content of each slide of your presentation in individual Markdown files; that is, each Markdown file corresponds to one slide of the presentation. Each Markdown file should (if applicable) contain the following information:
+
+* Type of slide (i.e. title slide, centered text, link + code, code centered, etc.) 
+* Title of slide
+* Text of slide
+* Images on slide
+* Code on slide
+* Speaker notes
+
+In these Markdown files, it is essential that you not only provide the literal text that will go on the slide, but also add any additional notes about formatting, organization, etc. so that when the files are to be converted to slides, it is clear to **anyone** exactly how the slide should be formatted. For example, in the Markdown file, bold words that should be bolded on the slide and indicate if a certain text should be highlighted, underlined, etc. in the final product.
+
+For details about placing your work in GitBook, see <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization#placing-your-work-in-gitbook">Placing your work in GitBook</a>.
+
+## Adding a Workshop in Collaboration with a Company
```

Comment:
for this section it was more about how a workshop collaboration with a company would get started, which entails steps like reaching out to developer advocates, setting up 1:1 calls where Bit's mission is introduced, propose content collaboration, then follow same workshop structure as "joining as an organization" link with exception that the devs looking to add a workshop have to make sure they meet with the company weekly to gather their input, as well as constantly follow-up with emails.

Reply:
sorry about the confusion!